package.json

```diff
@@ -7,6 +7,8 @@
     "build": "ng build",
     "watch": "ng build --watch --configuration development",
     "test": "ng test",
+    "test:e2e": "run-p start cypress",
```

Comment:
Could you please add a brief note to the project readme about these commands?

Reply:
Sure

Reply:
Thanks